lib/web/scripts/node-join/install.sh

```diff
@@ -987,6 +1004,21 @@ install_from_repo() {
     fi
 }
 
+install_from_updater() {
+    SCRIPT_URL="https://$TARGET_HOSTNAME:$TARGET_PORT/scripts/install.sh"
+    CURL_COMMAND="curl -fsl"
+    if [[ ${DISABLE_TLS_VERIFICATION} == "true" ]]; then
+        CURL_COMMAND+=" -k"
+        SCRIPT_URL+="?insecure=true"
+    fi
+
+    log "Requesting the install script: $SCRIPT_URL"
+    $CURL_COMMAND "$SCRIPT_URL" -o "$TEMP_DIR/install.sh" || (log "Failed to retrieve the install script." && exit 1)
+
+    log "Executing the install script"
+    bash "$TEMP_DIR/install.sh"
```

Comment:
```suggestion
    "$TEMP_DIR/install.sh"
```
This will use the version of bash available on `$PATH`, instead of `/bin/bash` as in the shebang.
I could see arguments for doing that (e.g., we tell people to invoke with `bash` directly), but want to make sure that's intentional.